tests/data_loading/test_augmenters.py

```diff
@@ -0,0 +1,81 @@
+from delira.data_loading import Augmenter, DataLoader, SequentialSampler
+from .utils import DummyDataset
+from ..utils import check_for_no_backend
+
+import unittest
+
+
+class TestAugmenters(unittest.TestCase):
+    def setUp(self) -> None:
+        self._dset_len = 500
+        self._batchsize = 3
+
+        if "drop_last" in self._testMethodName:
+            self._drop_last = True
+        else:
+            self._drop_last = False
+
+        dataset = DummyDataset(self._dset_len)
+        data_loader = DataLoader(dataset)
+        sampler = SequentialSampler.from_dataset(dataset)
+
+        if "parallel" in self._testMethodName:
+            self.aug = Augmenter(data_loader, self._batchsize, sampler, 2,
+                                 drop_last=self._drop_last)
+        else:
+            self.aug = Augmenter(data_loader, self._batchsize, sampler, 0,
+                                 drop_last=self._drop_last)
```

Comment:
I would prefer to create a function which can be called from the respective test where the arguments where `parallel` and `drop_last` can be set.

Reply:
This somehow is the function you're asking for :D

Reply:
I thought it would be nice to have it as a separate function with arguments and not by checking the function name ..

Reply:
Then I would propose not to do it in this PR (mainly because we are doing it the same way in other tests), but to do it during a test-refactoring.